zephyr/rz/rz_cfg/fsp_cfg/rzg/r_gpt_cfg.h

```diff
@@ -0,0 +1,20 @@
+/*
+* Copyright (c) 2020 - 2024 Renesas Electronics Corporation and/or its affiliates
+*
+* SPDX-License-Identifier: BSD-3-Clause
+*/
+#ifndef R_GPT_CFG_H_
+#define R_GPT_CFG_H_
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define GPT_CFG_PARAM_CHECKING_ENABLE (BSP_CFG_PARAM_CHECKING_ENABLE)
+#define GPT_CFG_OUTPUT_SUPPORT_ENABLE (1)
+#define GPT_CFG_WRITE_PROTECT_ENABLE (0)
+#define GPT_CFG_GPTCLK_BYPASS 1
```

Comment:
@nhutnguyenkc : Please recheck define for "GPT_CFG_GPTCLK_BYPASS" whether it is as your expectation or not.
As my checking, it always checks with "BSP_FEATURE_BSP_HAS_GPT_CLOCK", and current BSP_FEATURE_BSP_HAS_GPT_CLOCK = 0 for rzg3s.

Reply:
@thaoluonguw , I understand your concern. I have checked the MDF and see that the `1` is fixed for `GPT_CFG_GPTCLK_BYPASS` of RZ/G3S. I don't think I can change this value since it can make the driver malfunction.